File: Modul 6/soal 3/PRAK603-2410817210011-MuhammadDzulFathiAhyan.c
#include <stdio.h> 
int main()
{ 
    int r1, r2; 
    scanf("%d %d", &r1, &r2);
    int tabel1[r1], tabel2[r2];
    if(r1 != r2){printf("Jumlah tidak sama");return 1;} 
    for(int i = 0; i < r1; i++){scanf("%d", &tabel1[i]);} 
    for(int i = 0; i < r1; i++){scanf("%d", &tabel2[i]);}
    for(int i = 0; i< r1; i++){printf("%d ", tabel1[i] * tabel2[i]);} 
    return 0; 
}

File: Modul 6/soal 3/PRAK603-2410817210011-MuhammadDzulFathiAhyan.py
r1, r2 = map(int, input().split())
if r1 != r2:
    print("Jumlah tidak sama")
    exit()
ruang1 = list(map(int, input().split()))
raung2 = list(map(int, input().split()))
for i in range(r1):
    print(ruang1[i] * raung2[i], end=" ")